lib/db.js

```diff
@@ -125,7 +128,9 @@ module.exports = function (
                 authKey: keyFetchToken.authKey,
                 uid: keyFetchToken.uid,
                 keyBundle: keyFetchToken.keyBundle,
-                createdAt: keyFetchToken.createdAt
+                createdAt: keyFetchToken.createdAt,
+                tokenVerificationId: keyFetchToken.tokenVerificationId,
+                tokenVerified: keyFetchToken.tokenVerified
```

Comment:
IIRC the db-server won't accept `tokenVerified` as an argument when creating the token